NEWS.md

```diff
@@ -5,6 +5,15 @@ Python 3.6 support dropped
 
   afew stopped supporting the older python version 3.6.
 
+Handle DMARC report with empty spf or dkim XML nodes
+
+DMARC Filter: allow to define subject regexp
+
+Get notmuch database path using Database wrapper
```

Comment:
add "This allows FolderNameFilter to work with a relative path in database.path of notmuch config file". (as in the PR description of https://github.com/afewmail/afew/pull/321)

Reply:
Ok